build.gradle

```diff
@@ -20,3 +20,47 @@ project.ext {
         RELEASE_KEY_PASSWORD = "*****"
     }
 }
+
+// Copies unit test reports to the 'artifacts' directory
+task copyUnitTestBuildArtifacts << { task ->
+    def hashPipe = new ByteArrayOutputStream()
+    task.project.exec {
+        commandLine = ['git', 'rev-parse', '--verify', 'HEAD']
+        standardOutput = hashPipe
+    }
+
+    def date = new Date()
+    def formattedDate = date.format('yyyyMMdd-HHmmss')
+    def destDir = "artifacts/${formattedDate}"
+    task.project.exec {
```

Comment:
This was originally my fault, but I was thinking that maybe we should avoid 'commandLine' for anything that has native java APIs. That way any developers on windows can still build it.